modules/afamqp/afamqp.c

```diff
@@ -608,6 +683,10 @@ afamqp_dd_new(GlobalConfig *cfg)
 
   log_template_options_defaults(&self->template_options);
   afamqp_dd_set_value_pairs(&self->super.super.super, value_pairs_new_default(cfg));
+  afamqp_dd_set_ca_file((LogDriver *) self, "");
+  afamqp_dd_set_key_file((LogDriver *) self, "");
+  afamqp_dd_set_cert_file((LogDriver *) self, "");
+  afamqp_dd_set_peer_verify((LogDriver *) self, TRUE);
 
```

Comment:
I would rather use NULLs instead of empty values to indicate these options are unset.